src/main/java/com/flagsmith/FlagsmithClient.java

```diff
@@ -57,6 +58,16 @@ public void updateEnvironment() {
       // if we didn't get an environment from the API,
       // then don't overwrite the copy we already have.
       if (updatedEnvironment != null) {
+        List<IdentityModel> identityOverrides = updatedEnvironment.getIdentityOverrides();
+
+        if (identityOverrides != null) {
+          Map<String, IdentityModel> identityOverridesByIdentifier = new HashMap<>();
+          for (IdentityModel identity : identityOverrides) {
+            identityOverridesByIdentifier.put(identity.getIdentifier(), identity);
+          }
+          this.identityOverridesByIdentifier = identityOverridesByIdentifier;
+        }
```

Comment:
Similar comment here to the one I added to the node.js repo here: https://github.com/Flagsmith/flagsmith-nodejs-client/pull/143#discussion_r1470035008

Reply:
See [response](https://github.com/Flagsmith/flagsmith-nodejs-client/pull/143#discussion_r1471127301).

Reply:
As per discussion in Node.js client, let's update the name of this variable here too.

Reply:
Done.